docs/src/pages/landing-page/Index.vue

```diff
@@ -226,13 +213,45 @@ export default defineComponent({
   border-bottom: 1px solid $lp-light;
 }
 
-.support {
-  height: 100vh;
-}
-
 .social-channels-call-to-action {
   // TODO: reduce image height (from top), and remove this negative margin
   margin-top: -30%;
 }
 
+.why-quasar-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 16px;
+  margin: 0 auto;
+  width: 80%;
+}
+
+.astronaut-hand {
+  z-index: 0; // ensure image is always displayed behind text
+  width: 53%;
+  position: absolute;
+
+  &--left {
+    left: 0;
+    bottom: 0;
+
+    @media screen and (max-width: $breakpoint-sm-max) {
+      top: 60%;
```

Comment:
I'm afraid we'll need something a bit more elaborated to make the background work correctly into all viewports, but I'll leave this to Nick so he can tell you how does he want the background to behave when changing viewport

Reply:
I actually spoke with Nick about this during our call, and he asked that I make the astronaut hands move down on small screen sizes below  the 'consult tiers' button. Perhaps the result is not perfect, I have sent some screenshots to Nick to get his opinion.

Reply:
Mmm... Not sure I understood, I guess we can discuss this when we do the meeting all together